### ISSUE TITLE
Add in User Profile Avatar to Dashboard Listing

### ISSUE THREAD

**Issue by CBPSC** [DNNCommunity/Dnn.CommunityDashboard#1]
It would "humanize" and make the dashboard more visual to add in the user's avatar in the dashboard listing and pop-up.

**Comment by WillStrohl** [comment]
@CBPSC  Will this be your first code contribution to the project too?  😉  🤞

**Comment by CBPSC** [comment]
Perhaps, willing to share wisdom with the locals? lol!

**Comment by david-poindexter** [comment]
I don't think this is the correct repo - I'm am pretty sure it is https://github.com/DNNCommunity/Dnn.CommunityMetrics
